docsets/OKD/docset.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "OKD",
+    "version": "4.11",
+    "archive": "OKD.tgz",
```

Comment:
File paths are case-sensitive. You need to rename the archives to `OKD.tgz`.

Reply:
fixed